workflow/controller/informer/workflow_template_convert_test.go

```diff
@@ -14,23 +15,23 @@ import (
 func Test_objectToWorkflowTemplate(t *testing.T) {
 	t.Run("NotUnstructured", func(t *testing.T) {
 		v, err := objectToWorkflowTemplate(&corev1.Status{})
-		assert.EqualError(t, err, "malformed workflow template: expected \"*unstructured.Unstructured\", got \"*v1.Status\"")
+		require.EqualError(t, err, "malformed workflow template: expected \"*unstructured.Unstructured\", got \"*v1.Status\"")
 		assert.NotNil(t, v)
 	})
 	t.Run("MalformedWorkflowTemplate", func(t *testing.T) {
 		v, err := objectToWorkflowTemplate(&unstructured.Unstructured{Object: map[string]interface{}{
 			"metadata": map[string]interface{}{"namespace": "my-ns", "name": "my-name"},
 			"spec":     "ops",
 		}})
-		assert.EqualError(t, err, "malformed workflow template \"my-ns/my-name\": cannot restore struct from: string")
+		require.EqualError(t, err, "malformed workflow template \"my-ns/my-name\": cannot restore struct from: string")
 		if assert.NotNil(t, v) {
 			assert.Equal(t, "my-ns", v.Namespace)
 			assert.Equal(t, "my-name", v.Name)
 		}
```

Comment:
if I'm not mistaken, I think every instance of `if assert.*` can be changed to `require.*`. If the assertion fails, the test will be marked as failed, even if further assertions are not run due to the conditional. So that seems equivalent in all cases